backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -260,6 +263,29 @@ def get_documents_without_fulltext(self, data):
                 del document["_error"]
         return documents
 
+    def get_pdg_keywords(self, data):
+        keywords = data.get("keywords", [])
+        pdg_keywords = []
+
+        for keyword_entry in keywords:
+            # case when it's another schema - we continiue iteration
+            if 'schema' not in keyword_entry or keyword_entry['schema'] != 'PDG':
```

Comment:
it will raise key error if `schema` is not in keyword because with OR both conditions will be checked (in that case AND clause should be used).
However, you can avoid it by checking `if keyword_entry.get('schema') != 'PDG'` instead